packages/foreman/rubygem-os/rubygem-os.spec

```
@@ -0,0 +1,94 @@
+# Generated from os-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name os
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: Simple and easy way to know if you're on windows or not (reliably), as well as how many bits the OS is, etc
+Group: Development/Languages
+License: FIXME
```

```suggestion
License: MIT
```

https://github.com/rdp/os/commit/c5a7fa2ea8163095bb00a32ce4f0da2dee7be0b0 added the license metadata but looks like that hasn't been released.